cfme/scripting/setup_env.py

```diff
@@ -118,11 +118,16 @@ def setup_ha_env(cfme_version, provider, lease, desc):
 @click.option('--lease', default='3h', help='set pool lease time, example: 1d4h30m')
 @click.option('--sprout-poolid', default=None, help='Specify ID of existing pool')
 @click.option('--desc', default='Replicated appliances', help='Set description of the pool')
-def setup_replication_env(cfme_version, provider, lease, sprout_poolid, desc):
+@click.option('--remote-worker', is_flag=True, help='Add node to remote region')
```

Comment:
http://click.pocoo.org/5/options/#boolean-flags

Looking at the documentation of `is_flag` for click, the default is False. I think you should remove the default parameter statement `remote_worker=False` in the function definition below and just add the `remote_worker` option.

Reply:
Resolved with `d5d70ad`